### ISSUE TITLE
Unable to set path

### ISSUE THREAD

**Issue by sutex** [SCToolsfactory/SCJMapper-V2#91]
![2020-03-01 14_37_15-Settings](https://user-images.githubusercontent.com/10361359/75619305-2e81c780-5bca-11ea-9bf2-c2287af87b2a.png)
D:\Roberts Space Industries\StarCitizen\ is my path any variation or pointing to the live wont be accepted

In a another issue I just read, would also like to load a my current profile and re-map from that point. 
Is there any work around to do that?

**Comment by bm98** [comment]
Did you try to let the program find the path on its own? Uncheck the Path setting in Settings.

Then if the program does not find the path on its own you have to set it to the install Dir
i.e. D:\Roberts Space Industries
As you see it appends StarCitizen\LIVE to the path given in Settings and therefore you get twice the StarCitizen and this is likely wrong.

For the later - I don't read exported XMLs from the game as the format has changed over time and I would need to adjust across versions. May be it does not change that often anymore and then one could do an Import function. But once you start with an own profile it just stays so Import is a once used function and did not got any priority.. sorry for that one.

**Comment by sutex** [comment]
I unchecked it . And importing the exported xml. is going to be a thing. I know I need it.

**Comment by sutex** [comment]
Ran the new version and it showed me all my layout exports saved and SCJMapper-V2 is loading them. No-idea if this is now supposed to happening, but thanks :)

**Comment by bm98** [comment]
Intentionally...
ChangeLog: 
update Can read Exported XMLs from the Game (mostly ... max 3 points for curves are supported) (#91)

**Comment by sutex** [comment]
I saw the changlog last line later thanks heaps. As I'm new to all the Hotas stuff forever changing keys around in game. Now can export see what Ive done and add stuff Don't suppose there's an updated video a how to guide in the works, found one it's ok more would be better. I added you to a link here's the page feel free to elaborate  https://robertsspaceindustries.com/spectrum/community/SC/forum/50174/thread/star-citizen-graphical-reference-sheet-generator
